security/pkg/nodeagent/secretfetcher/secretfetcher.go

```diff
@@ -65,6 +63,14 @@ const (
 	// istioPrefix and prometheusPrefix are prefix in secrets generated by istio.
 	istioPrefix      = "istio"
 	prometheusPrefix = "prometheus"
+
+	// secretControllerResyncPeriod specifies the time period in seconds that secret controller
+	// resyncs to API server.
+	// example value format like "30s"
+	secretControllerResyncPeriod = "SECRET_WATCHER_RESYNC_PERIOD"
+
+	// ingressFallbackSecret specifies the fallback secret for ingress gateway.
+	ingressFallbackSecret = "INGRESS_GATEWAY_FALLBACK_SECRET"
```

Comment:
We need a way to configure this in Helm

Reply:
Added a TODO

Reply:
We need either: 
* a default secret fallback name (eg `istio-fallback`) and have this feature gated around the existence of the secret
* a helm flag

We definitely cannot have neither, but from our discussions earlier it sounds like we should go with the default secret fallback name